### PR TITLE
Clarification about Darwin by default

### PR DESCRIPTION
**Pull request by Kakadu**
A clarification what will happen if we run `npm install -g reason-cli` without version

See issue #377

### CODE REVIEW

File: docs/global-installation.md [hunk 1]
```diff
@@ -8,11 +8,13 @@ title: Global Installation
 
 | Platform  | Install command
 |-----------|-------------------------------------------------------------------------------------------------
-| **macOS**     | `npm install -g reason-cli@3.1.0-darwin`
+| **macOS**     | `npm install -g reason-cli` or `npm install -g reason-cli@3.1.0-darwin`
```
Comment (andreypopp) [inline docs/global-installation.md]: I don't think we should advertise `npm install -g reason-cli` but instead fix `latest` release tag not to point to Darwin build.

Comment (jaredly) [reply docs/global-installation.md]: what would latest point to?

Comment (andreypopp) [reply docs/global-installation.md]: Either:
- a dummy package which errors with a message to install a correct package
- a dummy package which installs the correct package during a postinstall phase (cc @jordwalke why we didn't do this, I can't remember)

**Comment by kevinsimper** [comment]
This can be closed now since the reason-cli is not mentioned anymore :)

cc. @chenglou

**Comment by chenglou** [comment]
Thanks

**Pull request closed without merging**